src/Magento/Framework/Escaper.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Magento\Framework;
+
+use Stringable;
+
+/**
+ * @phpstan-type CastableToString null|scalar|Stringable
```

Comment:
Should we replace `scalar` with a more explicit list: `int|float`?
Because `bool` is also a `scalar` and I'm not sure if we should accept this, because casting a bool to a string gives weird/maybe-unexpected output: https://3v4l.org/nU9Wq